src/app/components/LanguageInfo.svelte

```diff
@@ -76,6 +81,14 @@
     height: 200px;
   }
 
+  .copyright-field {
+    display: flex;
+    align-items: flex-end;
+    margin-bottom: 26px;
```

Comment:
Nit: we might want to prefer using rem over px according to our style guide: https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-prefer-rem-to-px-for-positioning

Reply:
Thanks, changed the other layout px to rem aswell